packages/components/organisms/f-form/src/components/_tests/Form.test.js

```diff
@@ -1,10 +1,89 @@
 import { shallowMount } from '@vue/test-utils';
-import VForm from '../Form.vue';
+import FForm from '../Form.vue';
+
+const formData = {
```

Comment:
It would be good to have some tests around the expected result of a consumer not setting the formData as expected. For example, what happens if they don't pass in a default `value`, `name` or a `label`? (Same for `buttonText`?)

Reply:
I've added a `formDataValidator` method now https://github.com/justeat/fozzie-components/pull/1554/files#diff-6129c1b3d5cfb74495e312f10baf699cd88ab3fa678749e2edde92146fab55eaR103 (and tests)